src/components/DetailsMenu/DetailsMenuCard/Toggles/WordWrapFormatToggle/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { usePreferencesAnalytics } from "analytics";
+import { WordWrapFormat } from "constants/enums";
+import { useLogContext } from "context/LogContext";
+import BaseToggle from "../BaseToggle";
+
+const WordWrapFormatToggle: React.FC = () => {
+  const { sendEvent } = usePreferencesAnalytics();
+  const { preferences } = useLogContext();
+  const { setWordWrapFormat, wordWrapFormat } = preferences;
+  const isChecked = wordWrapFormat === WordWrapFormat.Aggressive;
+
+  const onChange = (checked: boolean) => {
+    console.log("checked", checked);
+    if (checked) {
+      setWordWrapFormat(WordWrapFormat.Aggressive);
+      sendEvent({
+        format: WordWrapFormat.Aggressive,
+        name: "Toggled Word Wrap Format",
+      });
+    } else {
+      setWordWrapFormat(WordWrapFormat.Standard);
+      sendEvent({
+        format: WordWrapFormat.Standard,
+        name: "Toggled Word Wrap Format",
+      });
+    }
+  };
+  return (
+    <BaseToggle
```

Comment:
I know this would be introducing a new pattern, but did you consider using a radio group here? While toggle is definitely a reasonable choice, I feel like 1) the lengthy labels look a bit awkward compared to the on/off toggles, and 2) this setting isn't so much enabling something as choosing between two options. Fine with either though!